base_export_async/data/config_parameter.xml

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
There's a lot of code format changes in this commit.
Please, move them in a dedicated "black,  isort, prettier" commit.
Those doesn't seems to be related to copier.
Or am I mistaking?